src/components/PopupWithForm.js

```diff
@@ -27,12 +29,19 @@ export default class PopupWithForm extends Popup {
     return this._popupForm;
   }
 
```

Comment:
Please, make a `resetForm` method to reset the form in `index.js` in a `.then` block


```js

  resetForm() {
    this._popupForm.reset()
  }

```